helm/templates/phoenix/secret.yaml

```diff
@@ -7,6 +7,10 @@ type: Opaque
 data:
 {{- if .Values.auth.secret }}
 {{- range $authSecrets := .Values.auth.secret }}
+  {{- if eq $authSecrets.key "PHOENIX_DEFAULT_ADMIN_INITIAL_PASSWORD" }}
+  {{ $authSecrets.key }}: {{ (empty $authSecrets.value | ternary $.Values.auth.defaultAdminPassword $authSecrets.value) | b64enc }}
```

Comment:
Could you add to the NOTES.txt how to retrieve the value of the admin secret, for example pre-providing the `kubectl get secret ... | base64 -d` similar to bitnami